packages/components/psammead-caption/src/__snapshots__/index.test.jsx.snap

```diff
@@ -1,17 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`should render Caption with some offscreen text 1`] = `
-.c1 {
-  -webkit-clip-path: inset(100%);
-  clip-path: inset(100%);
-  -webkit-clip: rect(1px,1px,1px,1px);
-  clip: rect(1px,1px,1px,1px);
-  height: 1px;
-  overflow: hidden;
-  position: absolute;
-  width: 1px;
-}
```

Comment:
As spoke offline not sure how I feel about us losing this CSS across all our snapshots. Would be good to spend some time debugging it or at minimum opening a new issue to capture the loss of this to the reference `LgsrE`

Reply:
Thanks for noting this, Phil. The issue seems localised to importing the `@bbc/psammead-visually-hidden-text` component. It's used in `@bbc/psammead-brand` and the tests for `@bbc/psammead-caption`. 

I've looked through our repository and noted that these are the only two instances of a Psammead component pulling in another Psammead *component* (rather than utility).  Since we're not npm linking all Psammead packages prior to generating snapshots, I believe this is an expected consequence of the change.